progression/src/main/resources/progression-migrations/V2__Create_items.sql

```diff
@@ -25,3 +25,4 @@ INSERT IGNORE INTO items (Material, Namespace, Keyname, Name, ModelData, Glow, h
 
 -- Woodcutting-related
 INSERT IGNORE INTO items (Material, Namespace, Keyname, Name, ModelData, Glow, hasUUID) VALUES ('GLISTERING_MELON_SLICE', 'progression', 'tree_bark', '<aqua>Tree Bark', 1, 0, 0);
+INSERT IGNORE INTO items (Material, Namespace, Keyname, Name, ModelData, Glow, hasUUID) VALUES ('OAK_WOOD', 'progression', 'compacted_log', '<light_purple>Compacted Log', 0, 1, 0);
```

Comment:
This is my fault for taking so long to review, but could you please create a new migration sql file, e.g.

V3__Create_compacted_log_item.sql
and insert this line in there instead?

Same goes for the shop item migration script

Reply:
Maybe even like V2.001_blahblahblah.sql would work since its such a small change